BuildingBlocks/src/BuildingBlocks.Infrastructure/EventBus/RabbitMQ/EventBusRabbitMQ.cs

```diff
@@ -181,7 +181,9 @@ private async Task ProcessEvent(string eventName, string message)
 
         if (_subsManager.HasSubscriptionsForEvent(eventName))
         {
-            await using var scope = _autofac.BeginLifetimeScope(AUTOFAC_SCOPE_NAME);
+            var scope = await _logger.TraceTime(
```

Comment:
You can remove this. It doesn't take much time.

Reply:
Removed the unnecessary `TraceTime()` statement.